src/mkdocstrings/handlers/python.py

```diff
@@ -87,6 +90,12 @@ def update_env(self, md: Markdown, config: dict) -> None:  # noqa: D102 (ignore
         self.env.trim_blocks = True
         self.env.lstrip_blocks = True
         self.env.keep_trailing_newline = False
+        self.env.filters["brief_xref"] = self.do_brief_xref
+
+    def do_brief_xref(self, path: str) -> Markup:
+        """Filter to create cross-reference with brief text and full identifier as hover text."""
+        brief = path.split(".")[-1]
+        return Markup(f'<span data-autorefs-optional-hover="{path}">{brief}</span>')
```

Comment:
Both of these values need to be escaped, and I believe this is one way to achieve that:
```suggestion
        return Markup('<span data-autorefs-optional-hover="{}">{}</span>').format(path, brief)
```

Reply:
Do you mean we need to escape potential double-quotes in path and brief for example?
I think both format and the f-string give the exact same result :confused:

Reply:
```pycon
>>> from markupsafe import Markup
>>> path = 'foo&bar'
>>> brief = '"bar"'
>>> Markup(f'<span data-autorefs-optional-hover="{path}">{brief}</span>')
Markup('<span data-autorefs-optional-hover="foo&bar">"bar"</span>')
>>> Markup('<span data-autorefs-optional-hover="{}">{}</span>').format(path, brief)
Markup('<span data-autorefs-optional-hover="foo&amp;bar">&#34;bar&#34;</span>')
```

Reply:
Aaaaah my bad, I thought it was about the `format` *builtin*, but it's rather the `format` method of Markup. Thanks for the example.